File: app/src/main/java/cn/blinkdagger/androidLab/entity/TabItem.java
package cn.blinkdagger.androidLab.entity;

/**
 * 类描述：底部导航栏图标文字
 * 创建人：ls
 * 创建时间：2018/3/7
 * 修改人：
 * 修改时间：
 * 修改备注：
 */

public class TabItem {

    private String tabId;//Tab的唯一标识
    private int tabIndex;//Tab的唯一索引
    private String tabNormalIconName;//未选中状态图标名称
    private String tabCheckedIconName;//选中状态图标名称
    private String tabText;// 文字
    private String tabNormalTextColor;//未选中状态文字颜色
    private String tabCheckedTextColor;//选中状态文字颜色

}

File: app/src/main/java/cn/blinkdagger/androidLab/entity/TabItem.kt
package cn.blinkdagger.androidLab.entity

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * 类描述：底部导航栏图标文字
 * 创建人：ls
 * 创建时间：2018/3/7
 * 修改人：
 * 修改时间：
 * 修改备注：
 */
@Parcelize
class TabItem(
        val tabId: String? = null, //Tab的唯一标识
        val tabIndex : Int = 0, //Tab的唯一索引
        val tabNormalIconName: String? = null, //未选中状态图标名称
        val tabCheckedIconName: String? = null, //选中状态图标名称
        val tabText: String? = null, // 文字
        val tabNormalTextColor: String? = null, //未选中状态文字颜色
        val tabCheckedTextColor: String? = null, //选中状态文字颜色
) : Parcelable